### PR TITLE
aedt objects refactoring

### PR DESCRIPTION
**Pull request by maxcapodi78**
All AEDT objects are now initialized and managed in a specific class that can be run also standalone.
the objects are then used in all Pyaedt classes as part of Design initialization.

### CODE REVIEW

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM.

File: pyaedt/application/Design.py [hunk 1]
```diff
@@ -939,6 +721,54 @@ def default_solution_type(self):
...
+                count_consistent_designs = 0
+                for des in self.design_list:
+                    self._odesign = self._oproject.SetActiveDesign(des)
+                    if self._check_design_consistency():
+                        count_consistent_designs += 1
+                        activedes = des
+                if count_consistent_designs != 1:
+                    warning_msg = "No consistent unique design is present. Inserting a new design."
+                else:
+                    self.logger.info("Active Design set to {}".format(activedes))
```
Comment (akaszynski) [inline pyaedt/application/Design.py:764]: Nitpick: pyansys docs recommend [Use %-Formatting for Strings
](https://dev.docs.pyansys.com/guidelines/logging.html?highlight=logging#use-formatting-for-strings).

File: pyaedt/application/aedt_objects.py [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+        return None
+
+    @property
+    def o_symbol_manager(self):
+        """Aedt Simbol Manager.
+
+        References
+        ----------
+
+        >>> oSymbolManager = oDefinitionManager.GetManager("Symbol")"""
```
Comment (akaszynski) [inline pyaedt/application/aedt_objects.py:70]:
```suggestion
        >>> oSymbolManager = oDefinitionManager.GetManager("Symbol")
        """
```

File: pyaedt/application/aedt_objects.py [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+    @pyaedt_function_handler()
+    def get_module(self, module_name):
+        """Aedt Module object."""
+        if self.design_type not in ["EMIT"]:
+            return self.odesign.GetModule(module_name)
+        return None
+
+    @property
+    def o_symbol_manager(self):
+        """Aedt Simbol Manager.
```
Comment (akaszynski) [inline pyaedt/application/aedt_objects.py:65]:
```suggestion
        """AEDT Symbol Manager.
```

File: pyaedt/application/aedt_objects.py [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+
+        References
+        ----------
+
+        >>> oSymbolManager = oDefinitionManager.GetManager("Symbol")"""
+        return self.odefinition_manager.GetManager("Symbol")
+
+    @property
+    def opadstackmanager(self):
+        """Aedt oPadstackManager.
```
Comment (akaszynski) [inline pyaedt/application/aedt_objects.py:75]:
```suggestion
        """AEDT oPadstackManager.
```

File: pyaedt/application/aedt_objects.py [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+        return self._oboundary
+
+    @property
+    def oimport_export(self):
+        """Import/Export Manager Module.
+
+        References
+        ----------
+
+        >>> oDesktop.GetTool("ImportExport")"""
```
Comment (akaszynski) [inline pyaedt/application/aedt_objects.py:109]:
```suggestion
        >>> oDesktop.GetTool("ImportExport")
        """
```

File: pyaedt/application/aedt_objects.py [hunk 6]
```diff
@@ -0,0 +1,387 @@
...
+                self.design_type in ["Maxwell 3D", "Maxwell 2D"]
+                and self.odesign.GetSolutionType() == "Transient"
+                or self.design_type == "HFSS"
+            ):
+                self._omodel_setup = self.get_module("ModelSetup")
+        return self._omodel_setup
+
+    @property
+    def o_maxwell_parameters(self):
+        """AEDT Maxwel Parameter Setup Object.
```
Comment (akaszynski) [inline pyaedt/application/aedt_objects.py:211]:
```suggestion
        """AEDT Maxwell Parameter Setup Object.
```

File: pyaedt/application/Design.py (hunk 1 above)
Comment (maxcapodi78) [reply pyaedt/application/Design.py:764]: I need to check if this syntax is fully compatible with ironpython

**Review by Alberto-DM — APPROVED** [review]
Tested successfully. 
Great refactoring.

**Pull request merged**